test/surface/read.test.ts

```diff
@@ -247,5 +247,50 @@ describe('Read', () => {
     const result = await solidLogicBob.fetch(resourceUrl);
     expect(result.status).toEqual(403);
   });
-
+  it('Is disallowed to read .acl with Read', async () => {
+    const resourceUrl = `${testFolderUrl}9/test.txt`;
+    // This will do mkdir-p:
+    const creationResult =  await solidLogicAlice.fetch(resourceUrl, {
+      method: 'PUT',
+      body: '<#hello> <#linked> <#world> .',
+      headers: {
+        'Content-Type': 'text/turtle',
+        'If-None-Match': '*'
+      }
+    });
+    const aclDocUrl = await solidLogicAlice.findAclDocUrl(resourceUrl);
+    await solidLogicAlice.fetch(aclDocUrl, {
+      method: 'PUT',
+      body: makeBody('acl:Read', 'acl:Read', resourceUrl),
+      headers: {
+        'Content-Type': 'text/turtle',
+        // 'If-None-Match': '*' - work around a bug in some servers that don't support If-None-Match on ACL doc URLs
+      }
+    });
+    const result = await solidLogicBob.fetch(aclDocUrl);
+    expect(responseCodeGroup(result.status)).toEqual("4xx");
+  });
+  it('Is allowed to read .acl with Control+Read', async () => {
```

Comment:
Same as above, change from 'Control+Read' to 'Control'.

Reply:
Changing to Control only.